src/utils/formatNumber.ts

```diff
@@ -0,0 +1,8 @@
+const formatNumber = (number: number, options: object = {}): string => {
+  return new Intl.NumberFormat("en", {
+    maximumFractionDigits: 2,
+    ...options,
+  }).format(number);
+};
```

Comment:
1. заметил, что в коде в проекте ты прокидываешь в опции maximumFractionDigits и будто будет куда более явно определить значения по умолчанию сразу в аргументах функции (хотя в целом некрит)
2. лучше не отметать ts и задать для аргумента options тип опций. это даст тебе и строгость типизации, и удобство при разработке - тебе ide'шка сразу будет подсказывать, что можно засунуть в options

итого с этими всеми красивостями получается такое и вроде должно работать нормально:
```suggestion
const formatNumber = (
  number: number,
  { maximumFractionDigits = 2, ...rest }: Intl.NumberFormatOptions = {}
): string => {
  return new Intl.NumberFormat("en", {
    maximumFractionDigits,
    ...rest,
  }).format(number);
};
```